ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/services/representative-model-mapper.spec.ts

```diff
@@ -68,38 +92,50 @@ describe('RepresentativeModelMapper', () => {
         const expected = [HasAccessToCamera.Yes, HasAccessToCamera.No, HasAccessToCamera.NotSure];
 
         for (let i = 0; i < expected.length; ++i) {
-            givenAnswerIs(Keys.Camera, values[i]);
+            givenAnswerIs(RepresentativeQuestionKeys.Camera, values[i]);
             whenMappingModel();
             expect(model.camera).toBe(expected[i]);
         }
     });
 
     it('should map boolean values', () => {
-        givenAnswerIs(Keys.ClientAttendance, 'false');
-        givenAnswerIs(Keys.Computer, 'false');
-        givenAnswerIs(Keys.Room, 'false');
+        givenAnswerIs(RepresentativeQuestionKeys.ClientAttendance, 'false');
+        givenAnswerIs(RepresentativeQuestionKeys.Computer, 'false');
+        givenAnswerIs(RepresentativeQuestionKeys.Room, 'false');
+        givenAnswerIs(SelfTestQuestionKeys.SameComputer, 'false');
```

Comment:
We could (in a refactor PR) lift these tests that are both in rep and individual mapper specs to a spec for the base class to reduce duplication